challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -7761,6 +7761,724 @@
+            "assert.equal(luhnTest(\"1234567812345670\"),true,'<code>luhnTest(\"1234567812345670\")</code> should return <code>true</code>.');"
+        }
+      ],
+      "id": "5a23c84252665b21eecc7efd",
+      "challengeType": 5,
+      "releasedOn": "July 10, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": ["function luhnTest (str) {", "  // Good luck!", "}"],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "title": "Lychrel numbers",
+      "description": [
+        "<ol><li>Take an integer n, greater than zero.</li>",
+        "<li>Form the next n of its series by reversing the digits of the current n and adding the result to the current n.</li>",
+        "<li>Stop when n becomes palindromic - i.e. the digits of n in reverse order == n.</li></ol>",
+        "The above recurrence relation when applied to most starting numbers n = 1, 2, ... terminates in a palindrome quite quickly, for example if n<sub>0</sub> = 12 we get",
+        "12",
+        "12 + 21 = 33, a palindrome!",
+        "And if n<sub>0</sub> = 55 we get",
+        "55",
+        "55 + 55 = 110",
+        "110 + 011 = 121, a palindrome!",
+        "Notice that the check for a palindrome happens <i>after</i> an addition.",
+        "Some starting numbers seem to go on forever; the recurrence relation for 196 has been calculated for millions of repetitions forming numbers with millions of digits, without forming a palindrome. These numbers that do not end in a palindrome are called <b>Lychrel numbers</b>.",
+        "For the purposes of this task a Lychrel number is any starting number that does not form a palindrome within 500 (or more) iterations.",
+        "Seed and related Lychrel numbers:>",
+        "Any integer produced in the sequence of a Lychrel number is also a Lychrel number.",
+        "In general, any sequence from one Lychrel number <i>might</i> converge to join the sequence from a prior Lychrel number candidate; for example the sequences for the numbers 196 and then 689 begin:",
+        "196",
+        "196 + 691 = 887",
+        "887 + 788 = 1675",
+        "1675 + 5761 = 7436",
+        "7436 + 6347 = 13783",
+        "13783 + 38731 = 52514",
+        "52514 + 41525 = 94039",
+        "...",
+        "689",
+        "689 + 986 = 1675",
+        "1675 + 5761 = 7436",
+        "...",
+        "So we see that the sequence starting with 689 converges to, and continues with the same numbers as that for 196. Because of this we can further split the Lychrel numbers into true <b>Seed</b> Lychrel number candidates, and <b>Related</b> numbers that produce no palindromes but have integers in their sequence seen as part of the sequence generated from a lower Lychrel number.",
+        "Write a function that takes a number as a parameter. Return true if the number is a Lynchrel number else false. Iteration limit is 500."
```

Comment:
"Write a function that takes a number as a parameter. Return true if the number is a Lynchrel number. Otherwise, return false. The iteration limit is 500."